src/main/kotlin/io/kotlintest/Project.kt

```diff
@@ -1,59 +1,33 @@
 package io.kotlintest
 
-import org.reflections.Reflections
-import java.util.concurrent.atomic.AtomicBoolean
-import java.util.concurrent.atomic.AtomicInteger
-import kotlin.concurrent.thread
-
 object Project {
 
-  private fun discoverProjectConfig(): ProjectConfig? {
-    return if (System.getProperty("projectConfigScan") == "false") {
-      null
-    } else {
-      ReflectionsHelper.registerUrlTypes()
-      val configClasses = Reflections().getSubTypesOf(ProjectConfig::class.java)
-      when {
-        configClasses.size == 0 -> null
-        configClasses.size > 1 -> {
-          val configClassNames = configClasses.map { config -> config.simpleName }
-          throw InvalidConfigException("Multiple ProjectConfigs found: $configClassNames")
-        }
-        else -> configClasses.firstOrNull()?.kotlin?.objectInstance
+  private val ProjectConfigFQN = "io.kotlintest.provided.ProjectConfig"
```

Comment:
Lets start fields with a lower case letter. What is FQN? Better use a self-explanatory name.